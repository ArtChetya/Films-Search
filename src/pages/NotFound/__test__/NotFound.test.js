import { shallow } from 'enzyme/build';
import React from 'react';
import { NotFound } from '..';

describe('NotFound component', () => {
    let location;

    beforeAll(() => {
        location = {
            pathname: 'pathname'
        };
    });

    it('should be rendered correctly', () => {
        const component = shallow(<NotFound location={location} />);
        expect(component).toMatchSnapshot();
    });
});
