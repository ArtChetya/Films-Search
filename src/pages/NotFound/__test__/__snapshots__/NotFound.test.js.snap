// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should be rendered correctly 1`] = `
<span>
  Route 
  pathname
   has not been matched 
</span>
`;
